<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumbersWidget</class>
 <widget class="QWidget" name="NumbersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Numbers Formats</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QGroupBox" name="binaryGroup">
       <property name="title">
        <string>Binary</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QToolButton" name="binInButton">
          <property name="toolTip">
           <string>Press to import the selected binary number</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="binEdit">
          <property name="toolTip">
           <string>Enter the binary number</string>
          </property>
          <property name="maxLength">
           <number>64</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="binOutButton">
          <property name="toolTip">
           <string>Press to send the binary number to the current editor</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="octalGroup">
       <property name="title">
        <string>Octal</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QToolButton" name="octInButton">
          <property name="toolTip">
           <string>Press to import the selected octal number</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="octEdit">
          <property name="toolTip">
           <string>Enter the octal number</string>
          </property>
          <property name="maxLength">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="octOutButton">
          <property name="toolTip">
           <string>Press to send the octal number to the current editor</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="decimalGroup">
       <property name="title">
        <string>Decimal</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QToolButton" name="decInButton">
          <property name="toolTip">
           <string>Press to import the selected decimal number</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="decEdit">
          <property name="toolTip">
           <string>Enter the decimal number</string>
          </property>
          <property name="maxLength">
           <number>22</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="decOutButton">
          <property name="toolTip">
           <string>Press to send the decimal number to the current editor</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="hexGroup">
       <property name="title">
        <string>Hexadecimal</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QToolButton" name="hexInButton">
          <property name="toolTip">
           <string>Press to import the selected hex number</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="hexEdit">
          <property name="toolTip">
           <string>Enter the hex number</string>
          </property>
          <property name="maxLength">
           <number>16</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="hexOutButton">
          <property name="toolTip">
           <string>Press to send the hex number to the current editor</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="binTable">
     <property name="editTriggers">
      <set>QAbstractItemView::AllEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Input Format:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="E5TextSpinBox" name="formatBox">
       <property name="toolTip">
        <string>Select the input format</string>
       </property>
       <property name="wrapping">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Bitsize:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="E5TextSpinBox" name="sizeBox">
       <property name="toolTip">
        <string>Select the bit size</string>
       </property>
       <property name="wrapping">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="byteOrderButton">
       <property name="toolTip">
        <string>Press to swap the current byte order</string>
       </property>
       <property name="text">
        <string>Swap byte order</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>62</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>E5TextSpinBox</class>
   <extends>QSpinBox</extends>
   <header>E5Gui/E5TextSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>binInButton</tabstop>
  <tabstop>binEdit</tabstop>
  <tabstop>binOutButton</tabstop>
  <tabstop>octInButton</tabstop>
  <tabstop>octEdit</tabstop>
  <tabstop>octOutButton</tabstop>
  <tabstop>decInButton</tabstop>
  <tabstop>decEdit</tabstop>
  <tabstop>decOutButton</tabstop>
  <tabstop>hexInButton</tabstop>
  <tabstop>hexEdit</tabstop>
  <tabstop>hexOutButton</tabstop>
  <tabstop>binTable</tabstop>
  <tabstop>formatBox</tabstop>
  <tabstop>sizeBox</tabstop>
  <tabstop>byteOrderButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
